auditor/src/client/mod.rs

```diff
@@ -317,6 +325,8 @@ impl AuditorClient {
 /// The [`AuditorClientBlocking`] handles the interaction with the Auditor instances and allows one to add
 /// records to the database, update records in the database and retrieve the records from the
 /// database. In contrast to [`AuditorClient`], no async runtime is needed here.
+///
+/// It should be constructed using the [`AuditorClientBuilder`].
```

Comment:
Minor nitpick: "It is constructed using the..."

Reply:
Fixed